README.md

```diff
@@ -1,5 +1,16 @@
 # open-in-editor (Sourcegraph extension)
 
+## ⚠️ Deprecation notice
+
+**Sourcegraph extensions have been deprecated with the September 2022 Sourcegraph
+release. [Learn more](https://docs.sourcegraph.com/extensions/deprecation).**
+
+Starting with Sourcegraph 4.0, “Open in Editor” is a core part of the product. 🎉 Installing this extension is not needed and is not possible with 4.0+. For this reason, this extension is no longer in development.
+
+The repo and the old docs below are kept here to support older Sourcegraph versions.
+
+# Archive
+
```

Comment:
```suggestion
## ⚠️ Deprecation notice

**Sourcegraph extensions are deprecated with the September 2022 Sourcegraph
release. [Learn more](https://docs.sourcegraph.com/extensions/deprecation).**

## Description
```

The suggestion is to make the message generic and reuse it across all the extensions repositories. Information about what extensions migrated, what "deprecated" means, and how to still use the Sourcegraph extensions is on the linked docs page. 
What do you think?
cc: @vdavid, @philipp-spiess, @jjinnii, @ryankscott